src/devices/HuskyLens/README.md

```diff
@@ -0,0 +1,9 @@
+# HuskyLens
+HuskyLens is an easy-to-use AI machine vision sensor with 6 built-in functions: face recognition, object tracking, object recognition, line following, color detection and tag detection.
+
+Through the UART / I2C port, HuskyLens can connect to Arduino, Raspberry Pi, or micro:bit to help you make very creative projects without playing with complex algorithms.
+
```

Comment:
you may want to add a few more information explaining who to use properly this binding. And you can reuse the code you've been doing in the example.